runtime/gc_glue_java/ObjectModel.cpp

```diff
@@ -80,6 +80,8 @@ GC_ObjectModel::getSpecialClassScanType(J9Class *objectClazz)
 		result = SCAN_CLASSLOADER_OBJECT;
 	} else if ((NULL != _atomicMarkableReferenceClass) && isSameOrSuperClassOf(_atomicMarkableReferenceClass, objectClazz)) {
 		result = SCAN_ATOMIC_MARKABLE_REFERENCE_OBJECT;
+	} else if (objectClazz->gcPriorityScanField1 != 0) {
```

Comment:
should be `} else if (0 != objectClazz->gcPriorityScanField1) {`

Reply:
Removed - No longer needed